gatsby-node.js

```diff
@@ -162,14 +162,14 @@ exports.createPages = ({ graphql, actions }) => {
 		const numberOfPages = Math.ceil(posts.length / postsPerPage);
 
 		createPage({
-			path: `/`,
+			path: "/",
 			component: postList,
 			context: {
 				limitNumber: postsPerPage,
 				skipNumber: 0,
 				pageIndex: 1,
 				numberOfPages,
-				relativePath: ""
+				absolutePath: "/"
```

Comment:
the relative paths were not what the pagination component needed, so I updated them to absolute paths with slashes on both ends